src/parquet/reader-test.cc

```diff
@@ -15,12 +15,45 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+
 #include <gtest/gtest.h>
 
-namespace parquet {
+#include "parquet/reader.h"
+
+using std::string;
+
+namespace parquet_cpp {
+
+const char* data_dir = std::getenv("PARQUET_TEST_DATA");
```

Comment:
Good point. I'll add an entry to `setup_build_env.sh` and add something to the README